src/entities/EditorDocument/EditorDocument.spec.ts

```diff
@@ -1,6 +1,5 @@
 import { EditorDocument } from './index';
 import { BlockNode, BlockToolName, DataKey } from '../BlockNode';
```

Comment:
Looks like this test should not depend on BlockNode types and operate only with serialised values

Reply:
These dependencies are for types and assertions